packages/plugin/src/generators/e2e-project/e2e.ts

```diff
@@ -66,10 +66,21 @@ function validatePlugin(host: Tree, pluginName: string) {
 }
 
 function addFiles(host: Tree, options: NormalizedSchema) {
+  const projectConfiguration = readProjectConfiguration(
+    host,
+    options.pluginName
+  );
+  const { name: pluginPackageName } = readJson(
```

Comment:
Do we care if `pluginPackageName` is undefined? It is required in `package.json` so we could it's always there, otherwise it's a user error. We can also validate it and error out if it's missing somehow.

Reply:
I'll add this in a follow up.